src/components/Markdown/Headings.tsx

```diff
@@ -35,25 +36,29 @@ const HeadingLink: FunctionComponent<{
   id: string;
   level: number;
   title: string;
-}> = ({ id, level, title }) => (
-  <NavLink
-    _active={{ visibility: "initial" }}
-    _focus={{ visibility: "initial" }}
-    alignItems="center"
-    data-heading-id={`#${id}`}
-    data-heading-level={level}
-    data-heading-title={title}
-    display="flex"
-    id={id}
-    lineHeight={1}
-    opacity="hidden"
-    replace
-    to={`#${id}`}
-    visibility="hidden"
-  >
-    <LinkIcon boxSize={4} />
-  </NavLink>
-);
+}> = ({ id, level, title }) => {
+  const { search } = useLocation();
+
+  return (
+    <NavLink
+      _active={{ visibility: "initial" }}
+      _focus={{ visibility: "initial" }}
+      alignItems="center"
+      data-heading-id={`#${id}`}
+      data-heading-level={level}
+      data-heading-title={title}
+      display="flex"
+      id={id}
+      lineHeight={1}
+      opacity="hidden"
+      replace
+      to={`${search}#${id}`}
```

Comment:
Can we add a comment explaining what this means / what an example of this value `${search}#${id}` would be?

Reply:
Sure 👍